docs/reference/errors-and-warnings/NU1605.md

```diff
@@ -12,10 +12,35 @@ f1_keywords:
 
 # NuGet Warning NU1605
 
+## Example 1
+
 <pre>Detected package downgrade: 'PackageB' from 4.0.0 to 3.5.0. Reference the package directly from the project to select a different version.<br/>  'PackageA' 3.5.0 -> 'PackageB' 3.5.0<br/>  'PackageC' 4.0.0 -> 'PackageD' 4.0.0 -> 'PackageB' 4.0.0</pre>
 
 ### Issue
-A dependency package specified a version constraint on a higher version of a package than restore ultimately resolved. That is, because of the "nearest wins" rule when resolving packages, a nearer package in the graph may have overridden a distant package.
+A dependency package specified a version constraint on a higher version of a package than restore ultimately resolved. That is, because of the [`nearest-wins`](https://docs.microsoft.com/en-us/nuget/consume-packages/dependency-resolution#nearest-wins) rule when resolving packages, a nearer package in the graph may have overridden a distant package.
```

Comment:
```suggestion
A dependency package specified a version constraint on a higher version of a package than restore ultimately resolved. That is, because of the [nearest-wins](https://docs.microsoft.com/en-us/nuget/consume-packages/dependency-resolution#nearest-wins) rule when resolving packages, a nearer package in the graph may have overridden a distant package.
```